WordPress/Classes/Networking/Pinghub.swift

```diff
@@ -77,7 +86,14 @@ public class PinghubClient {
             guard let client = self else {
                 return
             }
-            client.delegate?.pinghubDidDisconnect(client, error: error)
+            let filteredError: NSError? = error.flatMap({ error in
+                // Filter out normal disconnects that we initiated
+                if error.domain == WebSocket.ErrorDomain && error.code == Int(WebSocket.CloseCode.normal.rawValue) {
+                    return nil
+                }
+                return error
+            })
```

Comment:
If I had written the web socket code, I would not have returned an error if we initiated the disconnect. Since patching WebSocket is trickier, I added this as a workaround as close to the socket as possible. The manager (delegate) shouldn't have to know about the specific implementation (`WebSocket`).